packages/@ember/canary-features/index.ts

```diff
@@ -18,6 +18,7 @@ export const DEFAULT_FEATURES = {
   EMBER_TEMPLATE_BLOCK_LET_HELPER: true,
   EMBER_METAL_TRACKED_PROPERTIES: null,
   EMBER_GLIMMER_ANGLE_BRACKET_INVOCATION: true,
+  EMBER_BASIC_TEMPLATE_HELPERS: true,
```

Comment:
The initial value here should be `null`. This helps ensure the tests pass with and without the feature. After the RFC is merged we will begin reviewing to “enable by default” (checking api docs, guides, implementation, etc) and then will flip from null to true.